lib/galaxy/webapps/galaxy/api/tools.py

```diff
@@ -110,6 +110,26 @@ def reload( self, trans, id, **kwd ):
 
     @expose_api
     @web.require_admin
+    def list_requirements(self, trans, **kwds):
+        """
+        GET /api/tools/list_requirements
```

Comment:
This is different from what is defined below in `lib/galaxy/webapps/galaxy/buildapp.py` , should be `/api/tools/all_requirements`